src/Flux/layers.jl

```diff
@@ -38,11 +38,13 @@ function diffeq_adjoint(p,prob,f,df,t,args...;kwargs...)
 end
 
 diffeq_adjoint(p::TrackedVector,args...;kwargs...) = Flux.Tracker.track(diffeq_adjoint, p, args...; kwargs...)
-
 # Example: https://github.com/JuliaDiffEq/DiffEqSensitivity.jl/blob/master/test/adjoint.jl
 # How to provide the cost function here? Is this fine?
 Flux.Tracker.@grad function diffeq_adjoint(p::TrackedVector,f,df,t,prob,args...;kwargs...)
   _prob = remake(prob,p=data(p))
   sol = solve(_prob,args...;kwargs...)
-  f(sol),Δ -> (Δ .* adjoint_sensitivities(sol,args...,df,t,kwargs...))
+  f(sol), Δ -> begin
+    grad = adjoint_sensitivities(sol,args...,df,t;sensealg=SensitivityAlg(autojacvec=false),kwargs...)
```

Comment:
`SensitivityAlg(autojacvec=false)` why this choice? I think we should just make this a kwarg as well, and add it in front.